### DISCUSSION TITLE
How to overcome the API limitation of 100 records?

### DISCUSSION THREAD

**Discussion by miletoda** [ramnes/notion-sdk-py#70] [Q&A]
When requesting data from the Notion database, which has about 500 records, the result is only 100, which is a limitation of the service's API. Do you use any way to load the full amount of data?

**Comment by mixeden** [comment]
You should pass special parameter to the request to get other rows: https://developers.notion.com/reference/pagination

**Comment by jheddings** [comment]
I have built paging iterators for pulling all content as needed.  See PR #51 or my new module that uses the SDK - [notional](https://github.com/jheddings/notional).
